ereolen_support/ereolen_support.form.inc

```diff
@@ -43,6 +43,10 @@ function ereolen_support_form($form, &$form_state) {
     [
       'type' => 'setting'
     ],
+    drupal_get_path('module', 'ereolen_support') . '/js/hideOptions.js',
+    [
+      'type' => 'setting'
```

Comment:
See https://github.com/eReolen/ereolen/pull/25 about using the correct type.

Reply:
Fixed